cron/poll_pull_requests.py

```diff
@@ -118,10 +119,18 @@ def poll_pull_requests():
                     old_votes[user] = 1
             json.dump(old_votes, fp)
 
+            # flush all buffers because we might restart, which could cause a crash
+            os.fsync(fp)
```

Comment:
@amoffat Are you suggesting that I directly use `JSONBackend`? Or that it is already used? Or that I should do something in the same vein?